usr.bin/netstat/netstat.1

```diff
@@ -401,6 +405,35 @@ See
 .El
 .It Xo
 .Bk -words
+.Nm netstat
+.Fl o
+.Fl 4 | Fl 6
+.Ek
+.Xc
+Either -4 (IPv4) or -6(IPv6) should be set. Print nexthop (nhops) information associated with routing entries.
```

Comment:
The introductory sentence should explain what it does. Switch the first two sentences, and combine the new second sentence with the third sentence.